### ISSUE TITLE
Export tagged PDFs for Accessibility

### ISSUE THREAD

**Issue by corey-slade** [puppeteer/puppeteer#7509]
In Chrome 85 an update was implemented to allow tagged PDFs to be generated, carrying over the alt text from the HTML to the PDF it creates. (https://blog.chromium.org/2020/07/using-chrome-to-generate-more.html#:~:text=Using%20Chrome%20to%20generate%20more%20accessible%20PDFs,-Wednesday%2C%20July%2029&text=Starting%20in%20Chrome%2085%20(scheduled,%22Save%20as%20PDF%22%20destination.)
For whatever reason when I generate a PDF from an HTML file using this package the tags are not carried over, whereas if I manually print to pdf from the same HTML file they are.

**Tell us about your environment:**

* Puppeteer version: puppeteer-core v10.0.0
* Platform / OS version: AWS Lambda
* URLs (if applicable): N/A
* Node.js version: NodeJS v14.x

**What steps will reproduce the problem?**
Generate a PDF from an HTML file containing alt tags on pictures, then check for tagged images in the resulting PDF. 
If generated manually from the Chrome browser it will carry over the alt tags into the PDF, if done with Puppeteer it fails to carry over the alt tags

**What is the expected result?**
The alt tags should be carried over into the resulting PDF, allowing us to comply with accessibility requirements

**What happens instead?**
The images are not tagged in the resulting PDF

**Comment by tjacobs3** [comment]
@corey-slade Until this gets built in, you can enable the chromium flag for exporting tagged pdfs with this

```
puppeteer.launch({ args: ['--export-tagged-pdf'] });
```

**Comment by corey-slade** [comment]
Hi @tjacobs3 thank you for your response, I tried adding the flag that way, but it doesn't work for some reason. When I generate a pdf manually in Chromium from the same HTML file it gives me a tagged PDF, but when I do it from my Lambda using Headless Chromium it doesn't carry over the tags

**Comment by madofo** [comment]
Could this issue be re-opened? I can confirm that the flag does not work as intended. Using Chromium print dialogue to generate PDF results in tagged pdf, but puppeteer generated PDF is not tagged.

**Comment by OrKoN** [comment]
That was limited by CDP previously but now it supports generateTaggedPDF in https://chromedevtools.github.io/devtools-protocol/tot/Page/#method-printToPDF

**Comment by Lightning00Blade** [comment]
Closed as completed by https://github.com/puppeteer/puppeteer/pull/11182 and release as of v21.4.0.
